docs/hadoop-catalog.md

```diff
@@ -79,12 +79,12 @@ Refer to [Schema operation](./manage-fileset-metadata-using-gravitino.md#schema-
 
 ### Fileset properties
 
-| Property name                                      | Description                                                                                            | Default value                | Required                                                                                   | Since Version   |
-|----------------------------------------------------|--------------------------------------------------------------------------------------------------------|------------------------------|--------------------------------------------------------------------------------------------|-----------------|
-| `authentication.impersonation-enable`              | Whether to enable impersonation for the Hadoop catalog fileset.                                        | The parent's (catalog) value | No                                                                                         | 0.6.0           |
-| `authentication.type`                              | The type of authentication for Hadoop catalog fileset, currently we only support `kerberos`, `simple`. | The parent's (catalog) value | No                                                                                         | 0.6.0           |
-| `authentication.kerberos.principal`                | The principal of the Kerberos authentication for the fileset.                                          | The parent's (catalog) value | required if the value of `authentication.type` is Kerberos and parent hasn't set the value | 0.6.0           |
-| `authentication.kerberos.keytab-uri`               | The URI of The keytab for the Kerberos authentication for the fileset.                                 | The parent's (catalog) value | required if the value of `authentication.type` is Kerberos and parent hasn't set the value | 0.6.0           |
+| Property name                                      | Description                                                                                            | Default value                 | Required | Since Version   |
+|----------------------------------------------------|--------------------------------------------------------------------------------------------------------|-------------------------------|----------|-----------------|
+| `authentication.impersonation-enable`              | Whether to enable impersonation for the Hadoop catalog fileset.                                        | The parent's (schema) value   | No       | 0.6.0           |
```

Comment:
`The parent's (schema) value` is also weird expression.
Maybe 
`The parent(schema)'s value`.
or
`The parent(schema) value`.
or 
`The parent schema value`.

I prefer option 3. Because I google key words. I find the usage of the third option.